docs/ops/config.rst

```diff
@@ -255,6 +255,16 @@ in other Django projects.
    If :envvar:`DJANGO_USE_OIDC` is set to ``True``, this settings must be set to
    the URL that a user can visit to logout. It may be a relative URL.
 
+.. envvar:: DJANGO_PEER_APPROVAL_ENFORCED
```

Comment:
My thinking is: If true, then we are enforcing the peer approval requirement; if false, we are not enforcing the peer approval requirement. Does that make sense?